feature/security/gnsi/credentialz/tests/hiba_authentication/README.md

```diff
@@ -0,0 +1,60 @@
+# Credentialz-5: Hiba Authentication
+
+## Summary
+
+Test that Credentialz properly configures (hiba) certificate authentication.
+
+
+## Procedure
+
+* Prior to writing test the following steps were taken to create appropriate keys/certs.
+  * Build hiba
+  * `./hiba-ca.sh -c` to create a CA keypair
+  * `./hiba-ca.sh -c -u -I testuser` to create a user key pair for the testuser user
+  * `./hiba-ca.sh -c -h -I dut` to create a new host key pair for the dut
+  * `./hiba-gen -i -f ~/.hiba-ca/policy/identities/prod domain example.com` to create a "prod" identity
+  * `./hiba-gen -f ~/.hiba-ca/policy/grants/shell domain example.com` to create a grant called "shell"
+  * `./hiba-ca.sh -p -I testuser -H shell` to give the "shell" grant to the testuser
+  * `./hiba-ca.sh -s -h -I dut -H prod -V +520w` to create a host certificate valid for a very long time (so we dont have to worry about it expiring and breaking the test)
+  * `./hiba-ca.sh -s -u -I testuser -H shell` to create a user certificate for testuser
+
+* Set DUT allowed authentication types to only public key using gnsi.Credentialz
+* Create a user `testuser` (with no certificate at this point)
+* Set the AuthorizedPrincipalsCommand by setting the tool to `TOOL_HIBA_DEFAULT`
+
+* Perform the following tests and assert the expected result:
```

Comment:
Maybe I don't know much about how we intend to break down these featureprofile tests - but I don't understand what this new README.md provides over the case in the last one. More explicit, perhaps, but is that what we want?

Reply:
from what ive seen there's just always a readme per test case/dir so I just yoinked stuff outa the original one into each of the others. I probably tweaked some things while doing so to make it make more clear for me (which... maybe made things worse :p), but we can obviously just copy/pasta back from the original or delete this and leave the original, just holler w/ whatever is the "right" thing to do here.